examples/tldraw-example/src/multiplayer-with-images/multiplayer.tsx

```diff
@@ -43,7 +43,6 @@ function Editor({ roomId }: { roomId: string }) {
          * without provifing a storage bucket based solution will cause
          * massive base64 string to be written to the liveblocks room.
          */
-        disableAssets={true}
```

Comment:
As mentioned in the comment, enabling assets without providing a storage bucket based encoding will cause massive base64 string to be written to Liveblocks. In my experimentation, love blocks can’t do large amount of data and will refuse to write videos (and very large images) as base64 outright.

Reply:
👍 I've kept this on a hidden route (#/multiplayer-with-images) and using small png's to work around this limitation.

Will leave it up to @steveruizok to do the hosted image implementation :)